packages/graphiql-toolkit/src/create-fetcher/types.ts

```diff
@@ -28,6 +28,7 @@ export type FetcherParams = {
   query: string;
   operationName?: string | null;
   variables?: any;
+  extensions?: any;
```

Comment:
This is the key line which adds `extensions` to the payload. Everything else in this PR is for the Extensions UI tab